### PR TITLE
Document HasSortableRelations trait

### PR DESCRIPTION
**Pull request by mjauvin**
Related: https://github.com/wintercms/storm/pull/94

### CODE REVIEW

File: database-traits.md [hunk 1]
```diff
@@ -676,3 +677,21 @@ class Product extends Model
     protected $nullable = ['sku'];
 }
 ```
+
+<a name="hassortablerelations"></a>
+## HasSortableRelations
+
+Add this trait to your model in order to allow its relations to be sorted/reordered.
+
+
```
Comment (LukeTowers) [inline database-traits.md:686]:
```suggestion
```

File: database-traits.md [hunk 2]
```diff
@@ -11,6 +11,7 @@
 - [Validation](#validation)
 - [Soft deleting](#soft-deleting)
 - [Nullable](#nullable)
+- [HasSortableRelations](#hassortablerelations)
```
Comment (LukeTowers) [inline database-traits.md:14]: Should be put next to Sortable and url should be `#has-sortable-relations`

Comment (mjauvin) [reply database-traits.md]: Done @LukeTowers

**Comment by mjauvin** [comment]
Can this be merged or are we waiting on something?

**Comment by LukeTowers** [comment]
@mjauvin did we already merge the related functionality?

**Comment by mjauvin** [comment]
we did

**Pull request merged**